Sources/Basics/AuthorizationProvider.swift

```diff
@@ -225,7 +225,7 @@ public class KeychainAuthorizationProvider: AuthorizationProvider, Authorization
         } catch {
             switch error {
             case AuthorizationProviderError.notFound:
-                self.observabilityScope.emit(info: "No credentials found for server \(server) in keychain")
```

Comment:
Why delete this?

Reply:
it is very noisy in the log, and I did not think it provided huge amount of value. we can also tune down to debug, or write code that caches negative results so it's called less often. opinions?

Reply:
I see. Maybe debug? If that's still too noisy then let's remove it.